tests/test_postgres_logical_replication.py

```diff
@@ -590,15 +799,21 @@ def test_run(self):
         menagerie.verify_sync_exit_status(self, exit_status, sync_job_name)
 
 
-        self.assertEqual(len(records_by_stream['postgres_logical_replication_test']['messages']), 2)
-        #first record will be the previous delete
+        self.assertEqual(len(records_by_stream['postgres_logical_replication_test']['messages']), 3)
+        #first record will be the previous first delete
         delete_message = records_by_stream['postgres_logical_replication_test']['messages'][0]
         sdc_deleted_at = delete_message['data'].get('_sdc_deleted_at')
         self.assertIsNotNone(sdc_deleted_at)
-        self.assertEqual(delete_message['data']['id'], 3)
+        self.assertEqual(delete_message['data']['id'], 4)
 
-        #second record will be the new update
-        update_message = records_by_stream['postgres_logical_replication_test']['messages'][1]
+        #second record will be the previous second delete
+        delete_message = records_by_stream['postgres_logical_replication_test']['messages'][1]
+        sdc_deleted_at = delete_message['data'].get('_sdc_deleted_at')
+        self.assertIsNotNone(sdc_deleted_at)
+        self.assertEqual(delete_message['data']['id'], 5)
```

Comment:
Oh, this is interesting. So the inclusive syncing caused the whole transaction to be re-replicated